templates/basic/.github/workflows/size.yml

```diff
@@ -0,0 +1,15 @@
+name: "size"
```

Comment:
```suggestion
name: size
```

YAML doesn't require quoting for the most part and the rest here and in the other actions in existing templates isn't quoted, so let's stay consistent

Reply:
Agree 👍